sql/core/src/test/resources/sql-tests/inputs/describe-query.sql

```diff
@@ -0,0 +1,28 @@
+-- Test tables
+CREATE table  desc_temp1 (key int COMMENT 'column_comment', val string) USING PARQUET;
+CREATE table  desc_temp2 (key int, val string) USING PARQUET;
+CREATE table  desc_temp3 (key int, val string) USING PARQUET;
```

Comment:
Shall we remove this 3rd `CREATE table desc_temp3`? That is only used at multi-insert test case, `FROM ... INSERT ... INSERT`. And, we can use like the following.
```
FROM desc_temp1 a
INSERT INTO desc_temp1 SELECT *
INSERT INTO desc_temp2 SELECT *;
```